SUMMARY.md

```diff
@@ -16,12 +16,12 @@
 * [Delivery Recipe](delivery_recipe/README.md)
     * [Risk Management](delivery_recipe/risk_management.md)
         * [Positive Risk Attitude](delivery_recipe/positive_risk_attitude.md)
-    * User Stories
```

Comment:
I think the pages need to be in their own section called 'Scrum', as we may have other methodologies later such as Kanban.